dev/builddeps-veloxbe.sh

```diff
@@ -84,9 +84,12 @@ fi
 
 ## compile gluten cpp
 cd $GLUTEN_DIR/cpp
-./compile.sh --build_velox_backend=ON --build_type=$BUILD_TYPE --build_velox_backend=ON \
-             --build_test=$BUILD_TESTS --build_benchmarks=$BUILD_BENCHMARKS --build_jemalloc=$BUILD_JEMALLOC \
-             --enable_hbm=$ENABLE_HBM --enable_s3=$ENABLE_S3 --enable_hdfs=$ENABLE_HDFS
+rm -rf build
+mkdir build
```

Comment:
```suggestion
mkdir build
cd build
```

@jinchengchenghh can you make a quick fix on this? thanks.